src/applications/vaos/tests/mocks/data.js

```diff
@@ -117,7 +117,10 @@ export function createMockAppointmentByVersion({
         sourceSystem: 'VCM',
         dateTime: fields.start,
         duration: fields.minutesDuration || null,
-        status: { description: null, code: 'FUTURE' },
+        status: {
+          description: null,
+          code: currentStatus || 'FUTURE',
```

Comment:
I think I would remove this change, probably. We always get FUTURE back as the status here from video appointments (which is wrong, but apparently the way it works).

Reply:
I'm sure I'm missing something but I can't get this to work without the change for canceled home vvs appointments. The code should be set to 'CANCELLED BY CLINIC'.

Reply:
Are we testing that kind of appointment somewhere? I don't think we actually get back canceled VVS appointments.

Reply:
I might be setting the test up wrong when I remove the change, it fails in CanceledAppointmentsList.unit.spec.jsx:233

Reply:
I might be setting the test up wrong when I remove the change, it fails in CanceledAppointmentsList.unit.spec.jsx:233

Reply:
Ah, so we are testing that. This is fine, then, it's not a big deal either way.